website/docs/r/interface_config.html.markdown

```diff
@@ -62,6 +62,10 @@ resource "aci_interface_config" "fabric_port_config" {
 * `description` - (Optional) The Description of the Port Configuration object. Type: String.
 * `annotation` - (Optional) The Annotation of the Port Configuration object. Type: String.
 * `name_alias` - (Optional) The Name Alias of the Port Configuration object. Type: String.
+* `operational_associated_group` - (Read-Only) The Distinguished Name of the Operational Associated Group.
+* `operational_associated_sub_group` - (Read-Only) The Distinguished Name of the Operational Associated Sub Group. This is only applicable when `port_type` is "access".
+* `port_dn` - (Read-Only) The Distinguished Name of the Port.
+* `pc_port_dn` - (Read-Only) The Distinguished Name of the Port Channel Port. This is only applicable when `port_type` is "access".
```

Comment:
We should add that this will contain the VPC Path DN when multiple ports from 2 switches are in the same Policy Group and that those 2 switches are in a VPC pair.

Reply:
Also mentioned that this path can be used with the static path resources. Same for port_dn

Reply:
okay